doc/classes/SpriteBase3D.xml

```diff
@@ -90,16 +93,19 @@
 			If set, lights in the environment affect the sprite.
 		</constant>
 		<constant name="FLAG_DOUBLE_SIDED" value="2" enum="DrawFlags">
-			If set, texture can be seen from the back as well, if not, it is invisible when looking at it from behind.
+			If set, texture can be seen from the back as well. If not, the texture is invisible when looking at it from behind.
 		</constant>
 		<constant name="FLAG_MAX" value="3" enum="DrawFlags">
 			Represents the size of the [enum DrawFlags] enum.
 		</constant>
 		<constant name="ALPHA_CUT_DISABLED" value="0" enum="AlphaCutMode">
+			This mode performs standard alpha blending. It can display translucent areas, but transparency sorting issues may be visible when multiple transparent materials are overlapping.
 		</constant>
 		<constant name="ALPHA_CUT_DISCARD" value="1" enum="AlphaCutMode">
+			This mode only allows fully transparent or fully opaque pixels. Harsh edges will be visible unless some form of screen-space antialiasing is enabled (see [member ProjectSettings.rendering/quality/screen_filters/screen_space_aa]). On the bright side, this mode doesn't suffer from transparency sorting issues when multiple transparent materials are overlapping. This mode is also known as [i]alpha testing[/i] or [i]1-bit transparency[/i].
 		</constant>
 		<constant name="ALPHA_CUT_OPAQUE_PREPASS" value="2" enum="AlphaCutMode">
+			This mode draws fully opaque pixels in a separate pass. This is slower than [constant ALPHA_CUT_DISABLED] or [constant ALPHA_CUT_DISCARD], but it allows displaying translucent areas and smooth edges while making sorting issues much less noticeable.
```

Comment:
I may be misunderstanding how opaque prepass works here, feel free to correct me.

Reply:
The opaque prepass allows the object to be drawn at the depth prepass stage. However, it uses an alpha scissor and so only draws the opaque parts of the object to depth.

Reply:
Is the description I wrote correct or should it be reworded?

Reply:
```suggestion
			This mode draws fully opaque pixels in the depth prepass. This is slower than [constant ALPHA_CUT_DISABLED] or [constant ALPHA_CUT_DISCARD], but it allows displaying translucent areas and smooth edges while using proper sorting.
```